lib/containers/common.pm

```diff
@@ -28,32 +28,48 @@ our @EXPORT = qw(install_podman_when_needed install_docker_when_needed allow_sel
   test_container_runtime test_container_image scc_apply_docker_image_credentials scc_restore_docker_image_credentials);
 
 sub install_podman_when_needed {
+    my $host_os = shift;
+    record_info $host_os;
+    my @pkgs = qw(podman);
     if (script_run("which podman") != 0) {
-        if (is_sle '>=15') {
-            add_suseconnect_product('sle-module-containers');
+        if ($host_os eq 'centos') {
+            assert_script_run "dnf -y install @pkgs", timeout => 160;
+        }
+        else {
+            if (is_sle '>=15') {
+                add_suseconnect_product('sle-module-containers');
```

Comment:
```suggestion
                add_suseconnect_product('sle-module-containers') if (is_sle '>=15');
```

Reply:
That was not my code but i will adjust it. more perlist

Reply:
I know :)  But since you are improving this tests, maybe some small thing to do :)